ipa/srs_precomp_test.go

```diff
@@ -11,14 +11,17 @@ import (
 )
 
 func TestPrecompSerde(t *testing.T) {
+	t.Parallel()
+
 	testNumPoints := []int{
-		2, 3, 7, 20, 64, // Arbitrary sizes
-		common.POLY_DEGREE, // Ethereum Verkle Tries case
```

Comment:
Downgrading a bit this setup which isn't adding much conscious coverage.
In any case, the important case is `common.POLY_DEGREE`.